sparsh2/CS-555-Senior-Sage
# SCRUM 49: Set up backend project structure

This PR sets up backend server project structure for development. Also adds a sample flask application and a test to kickstart development.

How to test:
- Ensure you are running this in a virtual environment
- Installation
  - Navigate to `backend/server/flaskr/`
  - Run `pip install -r requirements.txt`
  - Installation should be successful
- Run Flask app
  - From the same directory run `flask --app server run`
  - Server should start on local
- Run tests
  - A sample test has been included. Navigate to `backend/server/tests/`
  - Run `pytest`
  - Should pass 1 existing test
